internal/nix/nixprofile/profile.go

```diff
@@ -234,3 +257,28 @@ func ProfileRemoveItems(profilePath string, items []*NixProfileListItem) error {
 	}
 	return nix.ProfileRemove(profilePath, indexes)
 }
+
+// installableForPackage determines how nix profile should install this package.
+// Keeping in `nixprofile` package since its specific to how nix profile works,
+// rather than a general property of devpkg.Package
+func installableForPackage(pkg *devpkg.Package) (string, error) {
```

Comment:
Should this function just be part of devpkg.Package so that `URLForInstall` always returns the correct value?

We can rename `URLForInstall` if needed